app/lib/sms-games/controllers/SGCompetitiveStoryController.js

```diff
@@ -86,10 +90,17 @@ SGCompetitiveStoryController.prototype.createGame = function(request, response)
     return false;
   }
 
-  // Compile a new game document.
+  function truncName(name) {
+    if (!name) { return null; }
+    else if (stringValidator.containsNaughtyWords(name)) { return "IamNaughty"; }
```

Comment:
I'd push this `"IamNaughty"` up into its own module-wide var like `NAME_CHAR_LIMIT`. Will make it easier to find if we want to change it.